src/dotnet/commands/dotnet-help/xlf/LocalizableStrings.zh-Hant.xlf

```diff
@@ -253,8 +253,8 @@
         <note />
       </trans-unit>
       <trans-unit id="InstallDefinition">
-        <source>Add item to development environment.</source>
-        <target state="new">Add item to development environment.</target>
+        <source>Installs an item into the development environment.</source>
+        <target state="new">Installs an item into the development environment.</target>
```

Comment:
First, I don't know what I did to turn this yellow. No change or concern. 

Second, somebody tell me we have a tool that creates these and no one had to copy this change.

Reply:
We thankfully have a build task that generates the xlfs.